### PR TITLE
samples: event_manager_proxy: Adapt to nRF54h20 and sysbuild

### PR DESCRIPTION
**Pull request by KAGA164**
Align the event_manager_proxy sample with sysbuild and nrf54h20 support.

### CODE REVIEW

File: samples/event_manager_proxy/README.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ Configuration
 Selecting ICMSG backend
 =======================
 
-By default, the Event Manager Proxy sample uses the OpenAMP backend provided by the IPC Service.
+On nRF5340, by default, the Event Manager Proxy sample uses the OpenAMP backend provided by the IPC Service.
```
Comment (peknis) [inline samples/event_manager_proxy/README.rst:53]:
```suggestion
On the nRF5340 DK, this sample uses by default the OpenAMP backend provided by the IPC Service.
```

File: samples/event_manager_proxy/README.rst [hunk 2]
```diff
@@ -72,16 +74,32 @@ Complete the following steps to program the sample:
...
+
+      west build -b nrf54h20dk/nrf54h20/cpuapp -- -Devent_manager_proxy_SNIPPET=nordic-ppr
+
+   or use twister test case:
+
+   .. code-block:: console
+
+      west build -b nrf54h20dk/nrf54h20/cpuapp -T sample.event_manager_proxy.nrf54h20dk_cpuapp .
+
 #. Program both the cores:
```
Comment (peknis) [inline samples/event_manager_proxy/README.rst:96]:
```suggestion
#. Program both cores:
```

File: samples/event_manager_proxy/boards/nrf54h20dk_nrf54h20_cpuapp.overlay [hunk 3]
```diff
@@ -8,13 +8,9 @@
 /delete-node/ &ipc0;
 
 /* Enabled nodes required by IPC
- * Two mboxes, one for each sides and one ipc instance
+ * Mbox for ppr target, one for each sides and one ipc instance
```
Comment (peknis) [inline samples/event_manager_proxy/boards/nrf54h20dk_nrf54h20_cpuapp.overlay:11]:
```suggestion
 * Mbox for PPR target, one for each side and one IPC instance
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/event_manager_proxy/README.rst (hunk 1 above)
Comment (KAGA164) [reply samples/event_manager_proxy/README.rst]: Done

**Review by peknis — APPROVED** [review]

**Review by dchat-nordic — APPROVED** [review]

File: samples/event_manager_proxy/sample.yaml [hunk 4]
```diff
@@ -20,11 +20,10 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
     extra_args:
       FILE_SUFFIX=icmsg
-  sample.event_manager_proxy.nrf54h20dk_cpuapp.icmsg:
+  sample.event_manager_proxy.nrf54h20dk_cpuapp:
```
[the remark this answers is not in the document]
Comment (KAGA164) [reply samples/event_manager_proxy/sample.yaml:23]: It is set globally at the beginning of this file

[the remark this answers is not in the document]
Comment (maje-emb) [reply samples/event_manager_proxy/sample.yaml:23]: I had in mind the field `tags: sysbuild`.
?

**Review by grochu — APPROVED** [review]

**Pull request merged**